homesensorhub/communication/payload.py

```diff
@@ -0,0 +1,55 @@
+"""Module which implements the payload class."""
+import json
+
+
+class Payload:
+    """Class which implements the configuration of the payload for MQTT."""
+
+    def __init__(self, packet):
+        """Extract and structure the data from the packet."""
+        try:
+            self.__type = packet['type']
+            self.__name = packet['name']
+            self.__value = packet['value']
+            self.__timestamp = packet['timestamp']
+            self.__measurement = packet['measurement']
+        except KeyError as error:
+            print("The requested key is not available in the packet:\n{}"
+                  .format(error))
+
+    def get_mqtt_payload(self) -> json.dumps:
```

Comment:
This would make sense to be refactored as get_json_payload

Reply:
Fixed in 2330dc1ed3e118cfa8ff2ed43ea3d5ca33dd3422.